app/models/collection.ts

```diff
@@ -30,7 +30,7 @@ export default class CollectionModel extends OsfModel {
     @attr('array') statusChoices!: string[];
     @attr('array') volumeChoices!: string[];
 
-    @belongsTo('collection-provider', { inverse: 'collections' })
```

Comment:
`collection-provider` endpoint does not have a relationship named `collections`.